internal/iptables/iptables.go

```diff
@@ -234,7 +234,9 @@ func executeCommand(firewallConfiguration FirewallConfiguration, cmd *exec.Cmd)
 		// command.
 		//
 		// See https://github.com/rancher/k3s/issues/1434#issuecomment-629315909
-		args := append([]string{"--net", firewallConfiguration.NetNs, "--"}, cmd.Args...)
+		args := []string{fmt.Sprintf("--net=%s", firewallConfiguration.NetNs)}
+		args = append(args, "--")
+		args = append(args, cmd.Args...)
```

Comment:
```go
nsArgs := fmt.Sprintf("--net=%s", firewallConfiguration.NetNs)
args := []string{nsArgs, "--", cmd.Args...)
```

An alternative way to writing this that is a bit more concise. While it probably won't do much in terms of performance (we're working with small data here) I think it's nice to have the array already initialised.

I trust your judgement more than mine so I'll leave it up to you. TIOLI :D

Reply:
Is there another syntax for this?

```
internal/iptables/iptables.go:241:42: missing ',' in composite literal (typecheck)
                args := []string{nsArgs, "--", cmd.Args...}
```

Reply:
Oh I think I misread the code a little bit and gave a bad suggestion:

```
args := append([]string{nsArgs, "--"}, cmd.Args...)
```

this is what I had in mind

Reply:
Thanks! I resolved this and the question about the debug log line.